code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -1207,6 +1207,36 @@
 	if (prob(dose))
 		M.vomit()
 
+
+/datum/reagent/mental/kokoreed
+	name = "KoKo Reed Juice"
```

Comment:
is it KoKo or Koko, there are 3 different instances of it being spelled differently